internal/cmd/cli/console/connect/connect_test.go

```
@@ -0,0 +1,259 @@
+		Expect(err).To(HaveOccurred())
+		Expect(IsPermanentError(err)).To(BeFalse(), "Unavailable should be transient")
+		Expect(testSrv.connectCount.Load()).To(Equal(int32(1)))
+
+		// Second call: server returns Unavailable again.
+		err = connectOnce(ctx, opts, drainHandler)
+		Expect(err).To(HaveOccurred())
+		Expect(IsPermanentError(err)).To(BeFalse())
+		Expect(testSrv.connectCount.Load()).To(Equal(int32(2)))
+
+		// Third call: server succeeds (sends CONNECTED then DISCONNECTED, exits cleanly).
+		err = connectOnce(ctx, opts, drainHandler)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to err

(ineffassign)